src/api/TKRecoveryManager.m

```diff
@@ -0,0 +1,189 @@
+//
+//  TKRecoveryManager.m
+//  sdk
+//
+//  Created by Sibin Lu on 10/20/17.
+//  Copyright © 2017 Token Inc. All rights reserved.
+//
+
+#import "TKRecoveryManager.h"
+#import "TKClient.h"
+#import "TKUnauthenticatedClient.h"
+#import "TKCrypto.h"
+#import "TKLocalizer.h"
+#import "TKError.h"
+
+@implementation TKRecoveryManager {
```

Comment:
Do we need NSCoding or (de)serialization methods here so that the developer can save/load TKRecoveryManager when app is killed?

Reply:
good point but I would not suggest to do that for now.
1. The member recovery is usually finished in a short period. (Dont need to reload data most of the time)
2. The external developer will have to save the the serialization data somewhere and remember to set it to token io builder when restart the application.
3. If we have more than one manager in the future, It makes confused to the developers.

So I would suggest we support sterilization only if that's necessary and I dont think account recovery is the very moment.

Reply:
Ok it doesn't have to be part of this PR, but we should track it + make sure that we have it eventually since external developers will need to keep this state if they want to protect against the case where the app is killed.